angular-firebase/package.json

```diff
@@ -15,10 +15,12 @@
     "@angular/common": "~9.0.3",
     "@angular/compiler": "~9.0.3",
     "@angular/core": "~9.0.3",
+    "@angular/fire": "^5.4.1",
```

Comment:
You need either the firebase sdk js library or angular firebase library. There is no need to install both.